### PR TITLE
deprecating ssns for employees

### PR DESCRIPTION
**Pull request by Karlotcha**
Code changes: https://github.com/Gusto/zenpayroll/pull/70405

### CODE REVIEW

**Comment by Karlotcha** [comment]
I asked the question on slack, but posting it here too: 

> I am looking at https://docs.gusto.com/v1/employees and I believe this documents what you get from the API as a payroll admin. However, looking at the code, it looks like someone can access the API with employee-level permissions, and when this happens the schema is actually different? :thinkloaf:
What’s the employee-level API? Is it used by anyone? :thinkloaf:
See app/views/api/v1/employees/employee/show.json.rabl
(Fun fact: payroll admin API ssn field is called ssn, but employee API field is called hidden_ssn - both for employees records)

File: _posts/2013-08-07-employees.markdown [hunk 1]
```diff
@@ -19,7 +19,7 @@ title: Employees
 | `company_id`                  | Integer           |     X     |          |         | employee's company ID
 | `manager_id`                  | Integer           |     X     |          | null    | employee's manager ID
 | `department`                  | String            |     X     |          | null    | employee's department.
-| `ssn`                         | String            |           |    X     | null    | masked social security number. When updating this attribute, format should be a string of nine numbers (no dashes.)
+| `ssn`                         | String            |           |    X     | null    | Updates only (reads have been deprecated - this returns an empty string since July 2020). When updating this attribute, format should be a string of nine numbers (no dashes.)
```
Comment (DallasNChains) [inline _posts/2013-08-07-employees.markdown:22]: Should the default column say empty string instead of null here now?

**Review by DallasNChains — APPROVED** [review]
:shipit:

File: _posts/2013-08-07-employees.markdown (hunk 1 above)
Comment (Karlotcha) [reply _posts/2013-08-07-employees.markdown:22]: Ah yes I can change that! :)

**Pull request merged**